src/barcodes/Barcode.js

```diff
@@ -4,6 +4,10 @@ class Barcode{
 		this.text = options.text || data;
 		this.options = options;
 	}
+
+	static options() {
```

Comment:
I thought about it. The main reason why I wanted to make this a function is that it will make inheritance of options easier. If for all CODE128 codes have one option but we want to add additional options to that sub-symbology it can easily be done like follow.

````javascript
static options() {
  return Object.assign({}, super.options(), {
    somenewoptions: {}
  });
}
````